docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-fdr-host-inventory.md

```diff
@@ -0,0 +1,150 @@
+---
+id: crowdstrike-fdr-host-inventory-source
+title: CrowdStrike FDR Host Inventory
+sidebar_label: CrowdStrike FDR Host Inventory
+description: This document explains how to collect device information from the CrowdStrike FDR and send it to Sumo Logic.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The CrowdStrike FDR Host Inventory Source provides a secure endpoint to receive device data from the CrowdStrike [Host-And-Host-Group-Management-APIs](https://falcon.crowdstrike.com/documentation/84/host-and-host-group-management-apis#managing-hosts). It securely stores the required authentication, scheduling, and state tracking information.
+
+:::important
+The CrowdStrike API documentation is not public and can only be accessed by partners or customers.
+:::
+
+## Prerequisites
+
+To collect device data from the CrowdStrike platform, you must have an authorized CrowdStrike account. CrowdStrike APIs use an OAuth 2.0 authorization token to make authorized API calls. CrowdStrike API Client is required to get the OAuth 2.0 authorization token. To define a CrowdStrike API client, you must be designated as a `CrowdStrike Falcon Administrator` role.
+## Data sources
+
+The CrowdStrike FDR Host Inventory retrieves device information every 24 hours. The source will fetch device information that was seen in the last 30 days for the first time. Afterward, it will only fetch information of devices that have been modified.
```

Comment:
```suggestion
CrowdStrike FDR Host Inventory retrieves device information every 24 hours. The source will fetch device information that was seen in the last 30 days for the first time. Afterward, it will only fetch information about devices that have been modified.
```

Reply:
Done